linux_os/guide/services/xwindows/disabling_xwindows/xwindows_runlevel_target/ansible/shared.yml

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_sle,multi_platform_rhv,multi_platform_fedora,Red Hat Enterprise Linux 7,Red Hat Enterprise Linux 8
+# reboot = true
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: get default target
+  command: systemctl get-default
+  register: default_target
+- name: Switch to multi-user runlevel
+  command: systemctl set-default multi-user.target
+  when: default_target.stdout != "multi-user.target"
```

Comment:
This should use the `file` module and not the `command` module

Reply:
Do you mean to use the `file` module to create the symlink? Isn't there anything else that systemctl would do when setting the default target?

Reply:
No. Ansible expects you to use the `file` module here. It's an ansible ism that users will expect.

Reply:
done